spacewalk/setup/bin/cobbler20-setup

```diff
@@ -62,15 +69,19 @@ def main():
     gen_template(SETTINGS_TEMPLATE, answers, TEMP_SETTINGS)
     gen_template(MODULES_TEMPLATE, answers, TEMP_MODULES)
     copy(TEMP_SETTINGS, SETTINGS)
-    chmod(SETTINGS, 0644)
+    chmod(SETTINGS, 0o644)
     copy(TEMP_MODULES, MODULES)
-    chmod(MODULES, 0644)
-    m = re.compile(r"^cobbler\.host\s*=\s*", re.MULTILINE)
-
-    conf_lines = open(rhnConfig._CONFIG_FILE).read()
-    if not m.search(conf_lines):
-        system ("echo 'cobbler.host = %s' >> %s " % (answers['server'],
-                      rhnConfig._CONFIG_FILE))
+    chmod(MODULES, 0o644)
+
+    cobbler_host = None
+    with open(rhnConfig._CONFIG_FILE) as fh:
+        for line in fh.readlines():
+            if line.startswith('cobbler.host = '):
+                cobbler_host = True
+                break
+    if cobbler_host is None:
```

Comment:
Sorry, let's make it set to `False`, not to `None`. So then it is better to read:

```python
cobbler_host = False 
...
if not cobbler_host:
   ...
```

My bad, sorry.